### DISCUSSION TITLE
Starter module time consumption

### DISCUSSION THREAD

**Discussion by MihailTsybakov** [OpenRadioss/OpenRadioss#2808] [Q&A]
Hi @MarianBulla, @elequiniou! 

Im solving an explicit FEM problem with around 3 million nodes; Elements of type "solid" are used. I use elastic material model with thermal stress support, also several tie contacts are present in model. Im facing some strange behavior: for this problem the starter module takes around 4 hours, while the solution itself takes only 2. Why so and how can I fix that? Thanks in advance!

**Comment by roguic** [comment]
Hi @MihailTsybakov ,

I suspect a quadratic treatment in the code. If your model is not sensitive, can you please share it ?
My idea is to profile the starter and try to optimize the code to reduce the CPUTIME.

Best,
Romain
